scripts/release.sh

```diff
@@ -417,12 +418,21 @@ function parse_flags() {
     prepare_dot_release
   fi
 
+  if (( ! USE_KO_DOCKER_REPO )); then
+    # If they didn't use the special flag then force it to gcr
+    export KO_DOCKER_REPO="gcr.io/knative-nightly"
+  fi
+
   # Update KO_DOCKER_REPO and KO_FLAGS if we're not publishing.
   if (( ! PUBLISH_RELEASE )); then
     (( has_gcr_flag )) && echo "Not publishing the release, GCR flag is ignored"
     (( has_gcs_flag )) && echo "Not publishing the release, GCS flag is ignored"
-    KO_DOCKER_REPO="ko.local"
-    KO_FLAGS="-L ${KO_FLAGS}"
+    if (( USE_KO_DOCKER_REPO )); then
```

Comment:
Move the KO_DOCKER_REPO initialization to line 424, then here you just need to force it to ko.local if USE_KO_DOCKER_REPO is 0, like

```
(( ! USE_KO_DOCKER_REPO )) && KO_DOCKER_REPO="ko.local"
```

Reply:
okie dokie

Reply:
wasn't sure whether 424 was in the old vs new code.... check to see if I put it where you wanted